yts/test_suite_test.go

```diff
@@ -11,6 +13,29 @@ import (
 	"go.yaml.in/yaml/v4"
 )
 
+const (
+	ytsURL = "https://github.com/yaml/yaml-test-suite"
+	ytsTAG = "data-2022-01-17"
+	ytsDir = "testdata/" + ytsTAG
+)
+
+func TestMain(m *testing.M) {
+	if _, err := os.Stat(ytsDir); os.IsNotExist(err) {
+		log.Printf("Downloading YTS test suite into %s...", ytsDir)
+		err = exec.Command("git", "clone", "-q", ytsURL, ytsDir).Run()
+		if err != nil {
+			log.Fatalf("git clone failed: %v", err)
+		}
+		log.Printf("Checking out YTS tag %s...", ytsTAG)
+		err = exec.Command("git", "-C", ytsDir, "checkout", "-q", ytsTAG).Run()
+		if err != nil {
+			log.Fatalf("git checkout failed: %v", err)
+		}
+	}
+
+	os.Exit(m.Run())
```

Comment:
os.Exit is no longer expected in TestMain


```suggestion
	m.Run()
```